xm-commons-domainevent/src/main/java/com/icthh/xm/commons/domainevent/domain/DomainEventPayload.java

```diff
@@ -5,16 +5,17 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import java.io.Serializable;
 import java.util.Map;
 
 @Data
 @NoArgsConstructor
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "@type")
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "@type", defaultImpl = DomainEventPayload.class)
 @JsonSubTypes({
     @JsonSubTypes.Type(value = DbDomainEventPayload.class, name = "DbDomainEventPayload"),
     @JsonSubTypes.Type(value = HttpDomainEventPayload.class, name = "HttpDomainEventPayload")
 })
-public class DomainEventPayload {
+public class DomainEventPayload implements Serializable {
```

Comment:
what reason of Serializable?

in this case field date must be 

Map<String, ? extends Serializable>
and must be present serial version id

Reply:
removed